pkg/istio/authorizationpolicy/controller_test.go

```diff
@@ -217,7 +571,95 @@ func getFakeDomain() zms.SignedDomain {
 				{
 					Members:  []zms.MemberName{"productpage-reader"},
 					Modified: &timestamp,
-					Name:     domainName + ":role.productpage-reader",
+					Name:     domainNameOnboarded + ":role.productpage-reader",
```

Comment:
can we also create a `productpage-writer` role with 2 valid members, so that the created authz policy has a spec with source.Principals other than `*` and role name

Reply:
or better still get it added to the v2 provider unit tests instead

Reply:
makes sense to add it to the provide unit test